packages/lit-html/src/directives/live.ts

```diff
@@ -88,6 +100,14 @@ class LiveDirective extends Directive {
  * `live()` should not be used when the binding will cause a type conversion. If
  * you use `live()` with an attribute binding, make sure that only strings are
  * passed in, or the binding will update every render.
+ *
+ * To customize the equality check, you can pass in a function as the second
```

Comment:
I think the previous paragraph needs to be updated a bit to refer to that as default behavior.

Reply:
Updated. I'm not sure if the wording is clear enough, but I was not sure how to make it better.